src/main/java/sh/platform/config/Credential.java

```diff
@@ -1,12 +1,15 @@
-package sh.platform.config.reader;
+package sh.platform.config;
 
 import java.util.Collections;
 import java.util.Map;
 import java.util.Optional;
 
+/**
+ * An Unit  of a service that comes from {@link PlatformVariables#PLATFORM_RELATIONSHIPS}
```

Comment:
"A unit".  Yes, it begins with a vowel, but not a vowel *sound*, so no `n`.  English makes no sense.

That said, "a unit of a service" doesn't really make sense either.  "Access information for an individual service", maybe?